server/form-pages/apply/reasons-for-placement/basic-information/placementDate.ts

```diff
@@ -91,4 +91,16 @@ export default class PlacementDate implements TasklistPage {
 
```

Comment:
I think this commit message needs attention: 'As `noticeTypeFromApplication`
relies on data from the current page AND data from the current page, '

Reply:
Whoops!